src/tracks/playabletrack/wavetrack/ui/WaveformView.cpp

```diff
@@ -687,6 +687,7 @@ void DrawClipWaveform(TrackPanelDrawingContext &context, size_t channel,
    const double &t1 = params.t1;
    const double &averagePixelsPerSample = params.averagePixelsPerSample;
    const double &rate = params.rate;
+   const double &stretchRatio = 1.0;
```

Comment:
As I wrote on the closed PR, the commit "Make waveform display work again" doesn't completely do what it says.  Instead the next commit "Spectrum display..." completes the work for waveform display too.

So properly, these two commits should be squashed into one (plus the fixup I pushed).  Maybe that commit would be too large and you might find a nice way to separate small steps like changes of arguments of functions or construction of `ClipParameters`

Reply:
Right. I did not a good job with my interactive rebase for this part. I may mistakenly have attempted to re-order and squash the "WaveChannelView clean-up ..." into "Make waveform display work again", but actually it should have been squashed into "Spectrum display ..." (and then squashing that into "Make waveform display work again" as you suggested). I don't mind doing it, but is now a good time, since you've reviewed this already?